daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md

```diff
@@ -499,22 +496,28 @@ dapr run --app-id batch-sdk --app-port 7002 --components-path ../../../component
 The code inside the `process_batch` function is executed every 10 seconds (defined in [`binding-cron.yaml`]({{< ref "#componentsbinding-cronyaml-component-file" >}}) in the `components` directory). The binding trigger looks for a route called via HTTP POST in your Flask application by the Dapr sidecar.
 
 ```csharp
-Console.WriteLine("Processing batch..");
-string jsonFile = File.ReadAllText("../../orders.json");
-var ordersArray = JsonSerializer.Deserialize<Orders>(jsonFile);
-```
-
-The `batch-sdk` service uses the PostgreSQL output binding defined in the [`binding-postgres.yaml`]({{< ref "#componentbinding-postgresyaml-component-file" >}}) component to insert the `OrderId`, `Customer`, and `Price` records into the `orders` table. 
+app.MapPost("/" + cronBindingName, async () => {
 
-```csharp
-using var client = new DaprClientBuilder().Build();
+    Console.WriteLine("Processing batch..");
```

Comment:
I think in this small section we're trying to explain the cron binding, so I would remove all this file processing stuff.